infl-components-src/tooltip/index.js

```diff
@@ -59,7 +59,7 @@ export default class Tooltip extends Component {
   static propTypes = {
     title:             PropTypes.string,
     element:           PropTypes.node.isRequired,
-    position:          PropTypes.oneOf(['top', 'bottom']),
+    position:          PropTypes.string,
```

Comment:
It removes a stupid error that's not really an error. It could be a different PR, it just seems like a dumb PR.